Enigmatry.Entry.AspNetCore.SampleApp/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Enigmatry.Entry.AspNetCore.Exceptions;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Enigmatry.Entry.AspNetCore.SampleApp;
+
+#pragma warning disable CA1052
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        var builder = WebApplication.CreateBuilder(args);
+
+        var mvcBuilder = builder.Services.AddControllers();
+
+        ConfigureMvc(mvcBuilder, SampleAppSettings.Default());
+
+        var app = builder.Build();
+
+        app.AppUseExceptionHandler<Program>();
+
+        app.MapControllers();
+
+        app.Run();
+    }
+
+    internal static void ConfigureMvc(IMvcBuilder mvcBuilder, SampleAppSettings settings)
+    {
+        if (settings.UseNewtonsoftJson)
```

Comment:
Sample app can run in two modes - with SystemText.Json or Newtonsoft.Json. Until we migrate all projects to SystemText.Json we need to suport both modes :(